### ISSUE TITLE
Show worlds that are currently loading in Worlds facet

### ISSUE THREAD

**Issue by Hayden-da-Serg** [Neos-Metaverse/NeosPublic#2328]
## Is your feature request related to a problem? Please describe.
Without using the Legacy World Switcher, there's no way to tell if a world is loading if you started loading a world, then closed the current world while the opened world is still loading.

## Relevant issues
None to be found.

## Describe the solution you'd like
In the Currently Open Worlds, there should also be worlds that are in the process of loading to indicate that the world is actually loading.

## Describe alternatives you've considered
None that I could really think of.

## Additional context
A lot of experienced Neos users may have a facet on their home screen that shows only currently open sessions, having loading worlds fit in this category may be beneficial.

**Comment by shiftyscales** [comment]
This sounds as much like it's an issue in the display of the joining/connection text in that it sounds like it pre-emptively removes the joining text if you close your currently focused world?

**Comment by Hayden-da-Serg** [comment]
Yes, though I'd consider that to be a separate issue, and not necessarily the problem I'm trying to address, as this could also show if multiple worlds are loading (Something the Legacy world switcher can currently do)
